docs/oss/security/FasterXML/jackson-dataformats-text.md

```diff
@@ -89,21 +115,21 @@ Here is how the rating was calculated:
                     Here is how a number of watchers contributes to the score:
                     0 -> 0.00 (min), 450 -> 1.50, 750 -> 2.50, 3000 -> 10.00 (max)
       Importance:...Medium (weight 0.31 out of 1.00)
-      Value:........0.25  out of 10.00
+      Value:........0.26  out of 10.00
       Confidence:...10.00 out of 10.00
       Based on:...2 features:
-          Number of stars for a GitHub repository:......204
+          Number of stars for a GitHub repository:......208
           Number of watchers for a GitHub repository:...15
 
       Sub-score:....Vulnerability lifetime
       Importance:...Low (weight 0.23 out of 1.00)
-      Value:........3.25  out of 10.00
+      Value:........10.00  out of 10.00
       Confidence:...10.00 out of 10.00
       Based on:...2 features:
-          Info about vulnerabilities:...3 vulnerabilities
+          Info about vulnerabilities:...0 vulnerabilities
```

Comment:
Is this the case of ``FALSE POSITIVE` cases getting fixed?

Reply:
I think it should be 2 issues

https://www.cvedetails.com/vulnerability-list/vendor_id-15866/product_id-61565/Fasterxml-Jackson-dataformat-xml.html

Reply:
I think so. I could not find CVEs for jackson-dataformats-text.